pkg/report/report.go

```diff
@@ -98,6 +98,8 @@ var dateRe = regexp.MustCompile(`[a-z]{3}\d{1,2}`)
 // Map to handle RiskLevel -> RiskScore conversions.
 var Levels = map[string]int{
 	"harmless":   0,
+	"ignore":     0,
```

Comment:
I'd like to make sure that the API boundary is clear without any unanticipated surprises. The fact that the API returns dropped results would be a surprise to me.

What do you think about assigning the levels to be dropped by default (at least `ignore` and `none`) as `-1`? I'm iffy on `harmless`, it's probably OK as is.

Reply:
Makes sense. I like that.

Reply:
Updated in [`323a1e0` (#586)](https://github.com/chainguard-dev/malcontent/pull/586/commits/323a1e071a80e8508721031f29a89740fd03231e).